src/containers/common/ClassroomManagerContainer.jsx

```diff
@@ -40,6 +36,15 @@ class ClassroomManagerContainer extends React.Component {
     this.setState({ toast: { message: null, status: null } });
   }
 
+  deleteClassroom(id) {
+    // TODO: Add alert to ask if user is sure
+    Actions.deleteClassroom(id).then(() => {
+      // TODO: For API optimization, do we want to instead manually remove the classroom
+      // out of local app state instead of making another API call
```

Comment:
We can leave the comments in for now to remind ourselves of where optimization can happen, but you're right in that it's not critical for the initial launch of the app.